tests/copp/test_copp.py

```diff
@@ -22,30 +22,34 @@
         --pkt_tx_count <n> (int): How many packets to send during each individual test case.
             Default is 100000.
 
-        --swap_syncd: Used to install the RPC syncd image before running the tests. Default
-            is disabled.
+        --copp_swap_syncd: Used to install the RPC syncd image before running the tests. Default
+            is enabled.
 """
 
+import logging
 import time
 import pytest
 from collections import namedtuple
 
-from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
-from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]
 from tests.copp import copp_utils
 from tests.ptf_runner import ptf_runner
 from tests.common.system_utils import docker
-from tests.common.broadcom_data import is_broadcom_device
+
+# Module-level fixtures
+from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]
 
 pytestmark = [
-    pytest.mark.topology('t1')
+    pytest.mark.disable_loganalyzer,  # TODO: Figure out how to *only* ignore LLDP log messages.
```

Comment:
log analyzer was extended to support such a case in [qos](https://github.com/Azure/sonic-mgmt/blob/master/tests/qos/qos_sai_base.py#L463) test

Reply:
Sweet, fixed!